protocol/0081-SUCM-successor_markets.md

```diff
@@ -11,7 +11,7 @@ For [details of virtual stake calculation see how LPs are rewarded](./0042-LIQF-
 Many derivative markets would terminate and settle periodically but would be part of a lineage.
 Think e.g. of a [cash-settled future](./0016-PFUT-product_builtin_future.md) written on the same underlying that settles every three months.
 Successor markets are a feature that allows for markets to have a lineage, but most importantly allows LPs to keep their virtual stake built up on one market (parent) in the lineage to be transferred to the next one (successor).
-Moreover, part of the insurance pool of a parent market can be earmarked for transfer to the successor market instead of being distributed network wide (other markets in same settlement asset, global insurance pool).
+Moreover, part of the insurance pool of a parent market can be earmarked for transfer to the successor market instead of being distributed into the global insurance pool.
```

Comment:
```suggestion
Moreover, part of the insurance pool of a parent market can be earmarked for transfer to the successor market instead of being transferred into the global insurance pool.
```